### PR TITLE
Blank IntegerInput

### PR DESCRIPTION
**Pull request by casey1173**
Fixes a bug that did not allow for blank IntegerInputs, i.e., users could not backspace a digit to replace with another.
Also adds a help text pop up so users know the accepted range
Found no other Integer or Float inputs to fix--this seems to be the only one that had a looping value definition

### CODE REVIEW

**Review by casey1173 — COMMENTED** [review]
ignore this comment :-)

File: frontend/summary/summaryTable/studyEvaluationTable/Table.js [hunk 1]
```diff
@@ -338,10 +339,10 @@ class EditColumnForm extends Component {
                                         : store.settings.columns[colIdx].width;
                             store.updateStagedColumn(colIdx, {width});
                         }}
-                        value={col.width}
-                        required={true}
```
Comment (shapiromatron) [inline frontend/summary/summaryTable/studyEvaluationTable/Table.js:342]: @casey1173 @rabstejnek it looks like the only change is removing the required attribute. I guess if the width is unspecified, is there any way to assume the width is 1 for the SET table?

Comment (casey1173) [reply frontend/summary/summaryTable/studyEvaluationTable/Table.js:342]: it only updates the table width if the input is valid, plus when you create a table the initial width value is 1. so if a user removes the width val and presses update, the width will stay 1 (or whatever it was before)

**Review by rabstejnek — APPROVED** [review]

**Review by shapiromatron — APPROVED** [review]
nice, works well

**Pull request merged**